src/Faithlife.Data/DbConnectorCommand.cs

```diff
@@ -394,6 +405,9 @@ string Replacement(Match match)
 				command.Parameters.Add(dbParameter);
 			}
 
+			if (IsPrepared && !IsCached)
```

Comment:
```suggestion
			if (IsPrepared && !wasCached)
```

We want to prepare the command if it wasn't cached. `wasCached` should be set to `true` if and only if we get it out of the cache above.